rldb.py

```diff
@@ -34,7 +34,7 @@ def _generate_reactionrole_id(self):
         while True:
```

Comment:
By the way, this is not the proper way to proceed, that why we use [Primary Keys](https://www.sqlite.org/lang_createtable.html#rowid)